components/dashboard/src/App.tsx

```diff
@@ -26,19 +27,27 @@ const App: FC = () => {
     const history = useHistory();
     const location = useLocation();
     const search = useQueryParams();
+    const { isDark, setIsDark } = useTheme();
 
     useEffect(() => {
         const onKeyDown = (event: KeyboardEvent) => {
             if ((event.metaKey || event.ctrlKey) && event.key === "o") {
                 event.preventDefault();
                 history.push("/new");
+                return;
+            }
```

Comment:
Added this mostly out of desire to make this easier to test w/o constantly changing my OS theme. We can document it somewhere in the app if we end up thinking it's worth it.

Reply:
The shortcut seems nice. Have you tried changing the theme in the browser devtools? It's what I mostly use

<img width="1365" alt="image" src="https://github.com/gitpod-io/gitpod/assets/29888641/f3f80d4d-1f88-49ec-9a90-365c5f483073">

Reply:
Nice, that does work pretty well. It's probably pretty likely that this shortcut is going to conflict with something, somewhere, maybe I'll drop it in favor of just using that.